### PR TITLE
Allow multiple where queries

### PR DESCRIPTION
**Pull request by jamesbirtles**

### CODE REVIEW

**Review by connor4312 — APPROVED** [review]
Nice!

**Comment by connor4312** [comment]
Can we get a quick unit test for this?

File: uts.ts [hunk 1]
```diff
@@ -82,6 +82,15 @@ export abstract class Group {
         return this;
     }
 
+    protected getWhere(column: string): Comparator[] {
+        const clause = this._where[column];
+        if (!clause) {
+            return [];
+        }
+
+        return (Array.isArray(clause) ? clause : [clause]);
```
Comment (SimonSchick) [inline uts.ts:91]: redundant parens.

**Review by SimonSchick — APPROVED** [review]
One style nitpick, I'll leave that up to you.

**Pull request merged**